Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_RoomExtender.cs

```diff
@@ -0,0 +1,66 @@
+using UnityEngine;
+using System.Collections;
+
+public class VRTK_RoomExtender : MonoBehaviour
+{
+
+    public float additionalMovementMultiplier = 1.0f;
+    public float headZoneRadius = 0.25f;
+    public Transform debugTransform;
+    public Transform movementTransform;
+
+    protected Transform cameraRig;
+    protected Vector3 headCirclePosition;
+
+    void Start()
+    {
+        if (movementTransform == null)
+        {
+#if UNITY_5_4_OR_NEWER
```

Comment:
Look at this file:

https://github.com/thestonefox/SteamVR_Unity_Toolkit/blob/master/Assets/SteamVR_Unity_Toolkit/Scripts/Helper/DeviceFinder.cs#L35-L42

It has a method on it that you can use to find the headset, rather than doing it yourself.